src/components/chart/statistics-chart.ts

```diff
@@ -233,20 +233,18 @@ class StatisticsChart extends LitElement {
     const names = this.names || {};
     statisticsData.forEach((stats) => {
       const firstStat = stats[0];
+      const meta = this.statisticIds!.find(
+        (stat) => stat.statistic_id === firstStat.statistic_id
+      );
       let name = names[firstStat.statistic_id];
       if (!name) {
-        const entityState = this.hass.states[firstStat.statistic_id];
-        if (entityState) {
-          name = computeStateName(entityState);
-        } else {
-          name = firstStat.statistic_id;
+        const tmp: Record<string, StatisticsMetaData> = {};
+        if (meta) {
+          tmp[firstStat.statistic_id] = meta;
         }
+        name = getStatisticLabel(this.hass, firstStat.statistic_id, tmp);
```

Comment:
Let's change `getStatisticLabel` to accept just 1 meta item

Reply:
OK, I'll do that in a follow-up 👍